packages/code-studio/src/dashboard/DashboardContainer.jsx

```diff
@@ -276,6 +276,17 @@ export class DashboardContainer extends Component {
 
     layout.init();
 
+    await new Promise(resolve => {
+      if (layout.isInitialised) {
+        resolve();
+        return;
+      }
+      layout.on('initialised', () => {
+        log.debug('layout initialized');
+        resolve();
+      });
```

Comment:
Should probably remove the handler, just in case - 
```suggestion
      const onInit = () => {
        log.debug('layout initialized');
        layout.off('initialised', onInit);
        resolve();
      };
      layout.on('initialised', onInit);
```